### PR TITLE
Add stroke specs

### PR DESCRIPTION
**Pull request by mbasaglia**
Adds stroke, the schema is split to add gradient stroke in the future without duplicating all the properties.

### CODE REVIEW

File: schema/shapes/base-stroke.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                "ml": {
+                    "title": "Miter Limit",
+                    "type": "number",
+                    "default": 0
+                },
+                "ml2": {
+                    "title": "Miter Limit",
+                    "description": "Animatable alternative to ml",
+                    "$ref": "#/$defs/properties/scalar-property"
+                },
```
Comment (fmalita) [inline schema/shapes/base-stroke.json:29]: Oh, I did not know this was a thing, seems kinda confusing to have separate props for animatable vs. static.

Not the right time to change, but maybe we can revisit in the future where the "ml" property could be either static or animatable.  It should be fairly easy to discriminate at parse time, no?

Come to think of it, that approach could make sense for all non-keyframed animatable props, e.g.

```
"foo": $fooval
```
vs
```
"foo": { "a": 0, "k": $fooval}
```

The former is significantly more compact, and there are lots of non-keyframed props in all Lottie animations that would benefit.

File: schema/shapes/stroke-dash.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Stroke Dash",
+    "description": "An item used to described the dashe pattern in a stroked path",
```
Comment (fmalita) [inline schema/shapes/stroke-dash.json:5]: typo: "dash" or "dashed"?

**Review by fmalita — APPROVED** [review]

File: schema/shapes/base-stroke.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                "d": {
+                    "title": "Dashes",
+                    "description": "Dashed line definition",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/$defs/shapes/stroke-dash"
+                    }
+                }
+            },
```
Comment (Aidosmf) [inline schema/shapes/base-stroke.json:43]: the [lottie-docs/schema/base-stroke](https://lottiefiles.github.io/lottie-docs/schema/#/$defs/shapes/base-stroke) has `o` Opacity prop, are we not including it?

File: schema/shapes/stroke.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        {
+            "type": "object",
+            "properties": {
+                "ty": {
+                    "title": "Shape Type",
+                    "type": "string",
+                    "const": "st"
+                },
+                "c": {
+                    "title": "Color",
```
Comment (Aidosmf) [inline schema/shapes/stroke.json:22]: should we include a description as in [lottie-docs/schema](https://lottiefiles.github.io/lottie-docs/schema/#/$defs/shapes)?

```suggestion
                    "title": "Color",
                    "description": "Stroke color",
```

**Review by Aidosmf — APPROVED** [review]
LGTM, just a few questions

**Review by kudanai — APPROVED** [review]

**Pull request merged**